composer.json

```diff
@@ -45,6 +45,8 @@
         "jms/di-extra-bundle": "~1.7",
         "sensio/generator-bundle": "~2.3|~3.0",
         "symfony/yaml": "~2.3|~3.0",
+        "symfony/security-acl": "~2.4|~3.0",
+        "phpunit/phpunit": "~4.3",
```

Comment:
If someboday checks out SonataAdmin, this person should be able to run tests. For this, he need of course phpunit. But which version? Exactly the one that is used in composer.json of course!

That has nothing to do with "globally installed phpunit" or what ever. It doesnt bother me, where you got installed your phpunit... Put its a fact, that if I use another phpunit version than you, I cannot run correct tests...

At the end overall about this PR: It started on Nov **2014**. I added precision because your already written tests, relied on a precision of 8. I have covered FormBuilderIterator with a test (there was none) and I fixed a really bad code smell in FormBuilderIterator (accessing private properties over reflection that isnt even there in most of the implementations from other bundles).

Soooo..... And again reading your comments @greg0ire. Precision 8 is needed because tests are failiing otherwise (you rely on php.ini settings actually... those can be different of course...)

Commit-Message not correct? Ok, sorry... Maybe I find the time today, to fix this...